app/src/androidTest/java/com/example/pantrypal/LoginTest.java

```diff
@@ -0,0 +1,102 @@
+package com.example.pantrypal;
+
+import androidx.test.core.app.ActivityScenario;
```

Comment:
We've got a lot of unused imports here. You can run Ctrl + Alt + O to format the imports, which will also remove the unused ones.